crates/autopilot/src/arguments.rs

```diff
@@ -159,6 +159,15 @@ pub struct Arguments {
     /// is still considered for payout.
     #[clap(long, env, default_value = "5")]
     pub additional_deadline_for_rewards: usize,
+
+    /// Run the autopilot in a shadow mode by specifying an upstream CoW
+    /// protocol deployment to pull auctions from. This will cause the autopilot
+    /// to start a run loop where it performs solver competition on driver,
+    /// and report and log the winner **without** actually executing any
```

Comment:
The autopilot doesn't execute settlements in its normal mode neither, does it?

Reply:
No. I clarified this in the documentation.